base/components/LoginWidget.jsx

```diff
@@ -151,21 +152,37 @@ const RegisterLink = ({className, ...props}) => <LoginLink
 />;
 
 
+const RegisteredThankYou = () => {
+	const user = Login.getUser();
+	const name = user.name || user.xid;
+	return (<>
+		<h3>Thank you- and welcome!</h3>
+		Welcome to Good-Loop, {name}!<br/>
+		Check out your new account <a href="https://my.good-loop.com/#account">here</a>.
```

Comment:
LoginWidget gets used in app.sogive.org too. If we use {C.app.name} and the relative url `/#account` then this will probably work across apps.

Reply:
Done